conformance_pack/cloudfront.sp

```diff
@@ -229,16 +229,17 @@ query "cloudfront_distribution_use_secure_cipher" {
         aws_cloudfront_distribution,
         jsonb_array_elements(origins) as o
       where
-        o -> 'CustomOriginConfig' -> 'OriginSslProtocols' -> 'Items' @> '["TLSv1.2%", "TLSv1.1%"]'
+        o -> 'CustomOriginConfig' -> 'OriginSslProtocols' -> 'Items' @> '["TLSv1"]'
+        or o -> 'CustomOriginConfig' -> 'OriginSslProtocols' -> 'Items' @> '["SSLv3"]'
     )
     select
-      b.arn as resource,
+      distinct b.arn as resource,
```

Comment:
@ [sbldevnet](https://github.com/sbldevnet) Is this really required as we are using `distinct` in above CTE `origin_protocols`. ?

Reply:
Is it OK the current fix or should I remove the `origin_protocols` from the parent query to avoid the `distinct`?

Reply:
@sbldevnet We have tested the query below on our end, and it works fine even with two or more origins in the same distribution using the `SSLv3` and `TLSv1` protocols. Could you please confirm from your end and update the query so that we can proceed with the release?

Thanks.

```
with origin_protocols as (
  select
    distinct arn,
  o -> 'CustomOriginConfig' ->> 'OriginSslProtocols' as origin_ssl_policy
from
  aws_cloudfront_distribution,
  jsonb_array_elements(origins) as o
where
  o -> 'CustomOriginConfig' -> 'OriginSslProtocols' -> 'Items' @> '["TLSv1"]'
  or o -> 'CustomOriginConfig' -> 'OriginSslProtocols' -> 'Items' @> '["SSLv3"]'
)
select
  b.arn as resource,
  case
    when o.arn is null then 'ok'
    else 'alarm'
  end as status,
  case
    when o.arn is null then title || ' uses secure cipher.'
    else title || ' does not use secure cipher.'
  end as reason
from
  aws_cloudfront_distribution as b
  left join origin_protocols as o on b.arn = o.arn;
  ```

Reply:
@sbldevnet I am OK with the current fix. 👍